content/04.use-cases-b.md

```diff
@@ -37,17 +37,22 @@ For example, Creative Commons (CC; <https://creativecommons.org/licenses/>) lice
 A CC BY 4.0 license specifies that any code (or other creative products) must be appropriately credited to its original author when distributed, adapted or reused.
 
 
-### Collaborative and asynchronous code editing
+### Collaborative and asynchronous code editing{#code-editing}
 
 <!--*Contributors to this section: Kaitlyn Gaynor, Rob Crystal-Ornelas, Ali, Allison Binley*-->
 
-Researchers can asynchronously communicate and individually or collaboratively work in GitHub.
-By forking, branching, and cloning repositories ([Box 1](#definitions)), researchers can simultaneously work on different issues of the same research project to then later merge changes to the main branch project with pull requests.
-Explicit project organization and increased communication in GitHub Issues, Discussions, or within pull requests can help with project development and with potential merge conflicts due to users simultaneously working on the same sections [@doi:10.1007/s10664-019-09774-x].
-Moreover, version control [@https://sortee-github-hackathon.github.io/manuscript/v/latest/index.html#storing-and-archiving-version-controlled-data] and commit reversal features allow researchers to track and progress without worrying about irreparably modifying someone else's work.
+<!-- I think I accidentally added more jargon, but I feel like this section is really important to get right. I'd be open to leaving all the GitHub jargon out and simply citing a source on this instead.  -->
```

Comment:
great work, Eric! In the first sentence, I would change "share in-progress code" to "share in-progress work", to remove the focus on code. Similarly, in Line 52, maybe say "with the click of a button, greatly facilitating peer review" or add an "for example"